uitest/Configurations/AppConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+namespace DevHome.UITest.Configurations;
+
+/// <summary>
+/// Class model of the appsettings JSON file root object
+/// </summary>
+public class AppConfiguration
+{
+    /// <summary>
+    /// Gets or sets the target Dev Home package family name
```

Comment:
Maybe expand the comment to mention that the Dev/Canary/Release builds have different PFNs?